arcade_machine/components/carousel_menu.py

```diff
@@ -24,3 +24,9 @@ def set_to_zero_index(self):
             self.selected_item_index = 0
         except:
             pass
+
```

Comment:
You shouldn't EVER have a blanket except block, unless you re-raise the exception at the end.
I assume you are wanting to except a key error, which doesn't make a whole lot of sense either. Why are you passing invalid values into this function?